examples/playground/panel-lite-editor.js

```diff
@@ -0,0 +1,131 @@
+const appContainer = "app-container"
```

Comment:
## Todo

Add more features

- [ ] Ability to add list of requirements
- [ ] Ability to add list of environment variables. For example `OPENAI_API_KEY`. The environment variables should be added to `os.environ`.